app/lib/features/cross_signing/pages/request_cancelled.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:acter/common/themes/app_theme.dart';
+import 'package:atlas_icons/atlas_icons.dart';
+import 'package:flutter/material.dart';
+
+class RequestCancelledPage extends StatelessWidget {
```

Comment:
Nice to have these verification views being separately contained but we they can't be inferred as pages as they don't include ```Scaffold```, I would keep its outer directory as ```widgets/views``` and renaming them to replace ``View``` as ```RequestCancelledView```.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1257/commits/edb1022a67a1e0c339a3a546e3522ec47bdcf1e4